coding-concepts.js

```diff
@@ -40,7 +60,13 @@ var onlyOdds = [11, 12, 13, 14, 15].filter(value => value % 2 !== 0)
 // console.log(onlyOdds)
 
 // a) Your answer:
+// I'm expecting onlyOdds to be modified here but I THINK the date needs to be stored in another variable if it is to be retrieved. I'll say that the output will be the original array and not the filtered items due to not being assigned a value in a new variable.
+
 // b) Verify and explain:
+// Unfortunately for me, my second guess was correct, var is mutable and the new filtered data was reassigned to the declaration of onlyOdds and the assignment operator at the beginning.
```

Comment:
The filter method is just asking to output a new array that includes everything that is not divisible by two (odd). The console.log is just outputting the variable that is storing the results of the filter method.